### PR TITLE
RBTA-88/atambayev-daulet/navigation-bar/general

### PR DESCRIPTION
**Pull request by AtamanDaulet**
tests for Navigation Bar General. Please, check PR

### CODE REVIEW

**Comment by itrusevich** [comment]
1. Please resolve the conflicts
2. All the tests are working. But I think Artur recommended to use "assert" from chai instead of "expect". @artsenius is that ok to use expect?
3. I would make script titles more clear and removed the extra comments.

**Comment by AtamanDaulet** [comment]
I resolved conflict. I would appreciate if  you check again

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+import { expect } from 'chai';
+import nav from '../../../selectors/navigation-bar';
+import login from '../../../helpers/helpers'
+
+
+describe('Navigation Bar', function() {
+    it('exist Navigation Bar', function() {
+            login.login();
+            // browser.url('/');
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: remove comment

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            expect($(nav.navBar).isDisplayed()).to.eql(true);
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: please use assert

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    })
+
+    it('is contain 100% of the application width', function() {
+            let navBarWidth = parseInt($(nav.navBar).getCSSProperty('width').value);
+            let containerWidth = parseInt($(nav.container).getCSSProperty('width').value) - 30;
+            expect(navBarWidth).to.equal(containerWidth);
+        })
+        // Verify that Navigation  Bar located just below the Header.
+    it('located just below the Header', function() {
+            expect($(nav.navBar).getAttribute('class')).to.equal('card-footer text-center');
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: not sure if this test checking if it's really under the footer

**Comment by artsenius** [comment]
pull the latest changes from integration to see if there are still conflicts

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+import login from '../../../helpers/helpers'
+
+
+describe('Navigation Bar', function() {
+    it('exist Navigation Bar', function() {
+            login.login();
+            // browser.url('/');
+            expect($(nav.navBar).isDisplayed()).to.eql(true);
+        })
+        //'Navigation bar contains navigation buttons'
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: remove those comments. Leave just one-line spaces between tests.

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+            expect($(nav.logout).getCSSProperty('text-align').value).to.equal('center');
+        })
+        // Verify that Button Logout  have background color: #6c757d
+    it('have background color: #6c757d', () => {
+            expect($(nav.logout).getCSSProperty('background-color').parsed.hex).to.equal('#6c757d');
+        })
+        // Verify that Button Logout  have hover background color: #5a6268
+    it('have hover background color: #5a6268', () => {
+        $(nav.logout).moveTo();
+        browser.pause(2000);
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: too long pause. need to replace

File: wdio.conf.js [hunk 6]
```diff
@@ -53,13 +54,11 @@ exports.config = {
     // Sauce Labs platform configurator - a great tool to configure your capabilities:
     // https://docs.saucelabs.com/reference/platforms-configurator
     //
-    capabilities: [
-      {
+    capabilities: [{
```
Comment (artsenius) [inline wdio.conf.js]: Please do NOT modify config files except the place where you add specs

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+            expect($(nav.allIssues).getCSSProperty('text-align').value).to.equal('center');
+        })
+        // Verify that All Issues Button have background color: #17a2b8
+    it('have background color: #17a2b8', () => {
+            expect($(nav.allIssues).getCSSProperty('background-color').parsed.hex).to.equal('#17a2b8');
+        })
+        // Verify that All Issues Button  have hover background color: #138496
+    it('have hover background color: #138496', () => {
+            $(nav.allIssues).moveTo();
+            browser.pause(2000);
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: too long pause

File: test/specs/reg/navigationBar/general-nav-bar.js [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+            expect($(nav.newBug).getCSSProperty('text-align').value).to.equal('center');
+        })
+        // Verify that New Bug Button have background color: #17a2b8
+    it('have background color: #17a2b8', () => {
+            expect($(nav.newBug).getCSSProperty('background-color').parsed.hex).to.equal('#17a2b8');
+        })
+        // Verify that New Bug Button  have hover background color: #138496
+    it('have hover background color: #138496', () => {
+            $(nav.newBug).moveTo();
+            browser.pause(2000);
```
Comment (artsenius) [inline test/specs/reg/navigationBar/general-nav-bar.js]: too long pause

**Comment by AtamanDaulet** [comment]
corrected everything

**Pull request merged**